kasa/aestransport.py

```diff
@@ -41,6 +42,14 @@ def _sha1(payload: bytes) -> str:
     return sha1_algo.hexdigest()
 
 
+class AesState(Enum):
+    """Enum for AES state."""
+
+    HANDSHAKE = auto()  # Handshake needed
+    LOGIN = auto()  # Login needed
+    ESTABLISHED = auto()  # Ready to send requests
```

Comment:
Maybe `HANDSHAKE_REQUIRED` and `LOGIN_REQUIRED`, to make it completely clear what's the next step?

Reply:
ade408fe734fffaaf45fd7b6458cfff639e13e14